index.js

```diff
@@ -27,29 +38,84 @@ const urls = fs
   .split(/\r?\n/)
   .map((l) => `${l}`);
 
-const addonUUID = 'd56a5b99-51b6-4e83-ab23-796216679614';
+function whichBrowserIsSelected(isChromeSelected) {
+  return isChromeSelected
+    ? 'agbomdmjjfglgplpnkahbcmblehajkag'
+    : 'd56a5b99-51b6-4e83-ab23-796216679614';
+}
+let addonUUID = whichBrowserIsSelected(isChromeSelected);
 
-const options = new firefox.Options();
-options.setPreference(
-  'extensions.webextensions.uuids',
-  `{"firefox@ghostery.com": "${addonUUID}"}`,
-);
-// options.addArguments("--headless");
+let selectedBrowser = '';
+let options = ``;
+let outputPath = 'output/time';
+let driver = new Builder();
+let addon = '';
+let browserForDriver;
+
+if (isChromeSelected) {
+  selectedBrowser = 'Chrome';
+  options = new chrome.Options();
+  options.addArguments('--profile-directory=Default');
+  if (isGhosteryEnabled) {
+    options.addArguments(
+      `--user-data-dir=profiles/withGhostery/${selectedBrowser}`,
+    );
+    outputPath += `/withGhostery/${selectedBrowser}`;
 
-if (isGhosteryEnabled) {
-  options.addArguments('-profile', 'profiles/withGhostery');
-  outputPath += '/withGhostery';
+    console.log(`LOG: Installing addon for ${selectedBrowser}.`);
+    addon = await downloadAddon(chromeExtension);
+
+    options.addArguments(`--load-extension=${addon}`);
+  } else {
+    options.addArguments(
+      `--user-data-dir=profiles/withoutGhostery/${selectedBrowser}`,
+    );
+    outputPath += `/withoutGhostery/${selectedBrowser}`;
+  }
+  browserForDriver = Browser.CHROME;
+  driver = await new Builder()
+    .forBrowser(browserForDriver)
+    .setChromeOptions(options)
+    .build();
+} else if (isFirefoxSelected) {
+  selectedBrowser = 'Firefox';
+  options = new firefox.Options();
+  options.setPreference(
+    'extensions.webextensions.uuids',
+    `{"firefox@ghostery.com": "${addonUUID}"}`,
+  );
+  // options.addArguments("--headless");
+  if (isGhosteryEnabled) {
+    options.addArguments(
+      '-profile',
+      `profiles/withGhostery/${selectedBrowser}`,
+    );
+    outputPath += `/withGhostery/${selectedBrowser}`;
+
+    console.log(`LOG: Installing addon for ${selectedBrowser}.`);
+    addon = await downloadAddon(firefoxExtension);
+  } else {
+    options.addArguments(
+      '-profile',
+      `profiles/withoutGhostery/${selectedBrowser}`,
+    );
+    outputPath += `/withoutGhostery/${selectedBrowser}`;
+  }
+  browserForDriver = Browser.FIREFOX;
+  driver = await new Builder()
+    .forBrowser(browserForDriver)
```

Comment:
no need for variable 
```suggestion
    .forBrowser(Browser.FIREFOX)
```

Reply:
Done